### PR TITLE
Comments for alternate implementation in Adadelta Paper #36785

### PR DESCRIPTION
**Pull request by abhilash1910**
The original implementation of Adadelta by M.D Zeiler (https://arxiv.org/pdf/1212.5701.pdf) had a eta of 1.0 and epsilon of 1e-6 which is not matching with the function mentioned here. Hence commented the original implementation according to paper. Reference Issue #36785

### CODE REVIEW

File: tensorflow/python/keras/optimizer_v2/adadelta.py [hunk 1]
```diff
@@ -57,7 +57,8 @@ class Adadelta(optimizer_v2.OptimizerV2):
       ([pdf](http://arxiv.org/pdf/1212.5701v1.pdf))
 
   """
-
+  #def __init__(self, lr=1.0, rho=0.95, epsilon=None, decay=0., **kwargs):
```
Comment (tanzhenyu) [inline tensorflow/python/keras/optimizer_v2/adadelta.py]: No need for this.

File: tensorflow/python/keras/optimizer_v2/adadelta.py [hunk 2]
```diff
@@ -57,7 +57,8 @@ class Adadelta(optimizer_v2.OptimizerV2):
...
+  #Adadelta function definition as per paper by M.D. Zeiler (https://arxiv.org/pdf/1212.5701.pdf) where epsilon=1e-6and learning rate=1.0
```
Comment (tanzhenyu) [inline tensorflow/python/keras/optimizer_v2/adadelta.py]: Please put this at top of docstring.
Also please put description, i.e., which section in this paper that a recommended learning rate is proposed so users are well aware of it.

Comment (abhilash1910) [reply tensorflow/python/keras/optimizer_v2/adadelta.py]: Hi @tanzhenyu, sure I will put it at the top of the docstring.

**Comment by abhilash1910** [comment]
Hi @tanzhenyu,  raised a PR #36849.Thanks

**Comment by gbaned** [comment]
@abhilash1910 Can you please resolve conflicts? Thanks!

**Comment by abhilash1910** [comment]
@gbaned,yes I can resolve the conflicts.  I raised a new PR to resolve this issue #36849. Should I close this pr?

**Comment by tanzhenyu** [comment]
> @gbaned,yes I can resolve the conflicts. I raised a new PR to resolve this issue #36849. Should I close this pr?

Sure we can close it if you have another pr

**Comment by abhilash1910** [comment]
Yes @gbaned, thanks.

**Comment by abhilash1910** [comment]
Hi @tanzhenyu @gbaned can you check the PR #36849 so that if anything is required to be added, I check. Thanks.
